option.go

```diff
@@ -282,6 +302,61 @@ func (option *Option) Set(value *string) error {
 	return convert("", option.value, option.tag)
 }
 
+// Set the value of a "terminated" option to the specified value.
+// An error will be returned if the specified value could not be
+// converted to the corresponding option value type.
+// If option value is slice of slice, the parameter is converted
+// as a slice and appended to the option value. Otherwise, if
+// option value is slice, option value is replaced by parameter.
+func (option *Option) SetTerminatedOption(value []*string) error {
```

Comment:
Naming: Given this is on the `Option` struct, the `Option` suffix in the name is unnecessary (we don't have `SetOption`). So I think this should be called `SetTerminated`.

Reply:
Also: We only ever use `*val` below and don't support boolean options here, right? (That wouldn't make much sense.) So should the signature be just a slice of strings: `SetTerminated(value []string)`?

Reply:
Thanks!